examples/acquiring_data/querying_the_GOES_event_list.py

```diff
@@ -4,59 +4,67 @@
 ==================================================
 
 How to retrieve the GOES flare event list through use of
-SunPy's Heliophysics Event Knowledgebase (HEK) client.
+SunPy"s Heliophysics Event Knowledgebase (HEK) client.
```

Comment:
```suggestion
SunPy'"'s Heliophysics Event Knowledgebase (HEK) client.
```

Reply:
haha this is what I get for doing a blind find replace.